packages/ids-import/README.md

```diff
@@ -0,0 +1,11 @@
+# Apps-script code for IDS Project
```

Comment:
Please write out full IDS name and link to it

Reply:
This is not resolved - where is the link and where is the full name? I______ D______ S_____

Reply:
I was about to push it 😅